js/main.js

```diff
@@ -229,11 +229,16 @@ $(window).bind('keydown', function(e) {
         // Select all text in address bar
         $('.js-navAddressBar').select();
         break;
+      case config.keyShortcuts.save:
+        // Trigger any visible .saveForm button
+        // .last() will make sure you'll only click one button if 2 are visible (listing form)
+        $('.js-saveForm:visible').last().trigger('click');
```

Comment:
> or assume the only way to save a form is by clicking a button

This functionality is available only for **certain clickable** buttons (mostly main forms in user settings). If there are other forms that can be saved and don't use the same layout as the existing one, that's a bad UI design which should be fixed instead. There shouldn't be a situation when user needs to learn many different UI patters.

> Instead, fire an event on window.obEventBus

Why is the custom event handler better (for other than the custom name)? I think triggering existing click events (for clickable buttons) is a good solution and as a bonus it doesn't add any complexity to existing views.